src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -92,6 +95,12 @@ public ObservableList<Session> getFilteredSessionList() {
         return model.getFilteredSessionList();
     }
 
+    @Override
+    public ObservableList<AssignmentName> getAssignmentNameList() {
+        AssignmentInitialise.init();
+        return AssignmentNameInitialise.getAllNames();
```

Comment:
Could add a test case covering this line

Reply:
I agree. Codecov will be happier like this